### PR TITLE
Multiple Things, Read Description

### PR DESCRIPTION
**Pull request by remotevase**
1. Updates the Issue Template
2. Creates a PR Template
3. Moves the CONTRIBUTING.md file to its universal location in the .github folder
4. Well, nothing else, really...

### CODE REVIEW

File: .github/ISSUE_TEMPLATE [hunk 1]
```diff
@@ -0,0 +1,30 @@
+## **SUBMISSIONS LACKING A RESPONSE TO ONE OF THE FIELDS WILL BE DELETED**
```
Comment (SOF3) [inline .github/ISSUE_TEMPLATE]: Why are you using `##`? This will only be emphasized _after_ the issue is created.

File: .github/ISSUE_TEMPLATE [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+PocketMine version should include the Jenkins build number or the git commit hash.
+--->
+> PocketMine-MP Version:
+PHP Version:
+Server Operating System:
+Game Version- PE or Win10: 
+
+### Crash Dump, Backtrace, or any Other Information
+
+> Please use gist or https://www.filedropper.com/ and add the link here
```
Comment (SOF3) [inline .github/ISSUE_TEMPLATE]: And why are you advertising filedropper.com when we can use "anything else"?

File: .github/ISSUE_TEMPLATE [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+### Make sure all the checkboxes below are checked. If not, the issue will be deleted.
+<!--- The checkboxes can be checked AFTER the issue is submitted. -->
+- [ ] This issue isn't duplicated - check by using the search bar at the top of your screen, and selecting the "issues" tab.
+- [ ] This issue is written in english for clarity. Use https://translate.google.com to translate to english.
+- [ ] This issue does not use abbreviations- type out the full word or phrase please. (Understandability)
+- [ ] The cause of this issue is not because of a plugin
```
Comment (SOF3) [inline .github/ISSUE_TEMPLATE]: If people can read, they would read the contribution guidelines. Even if they don't, they won't look at new checkboxes after the issue is submitted. Would work better in the comments, but again, if they would read the small print, it is much easier to link them to contribution guidelines.

File: .github/ISSUE_TEMPLATE [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+Game Version- PE or Win10: 
+
+### Crash Dump, Backtrace, or any Other Information
+
+> Please use gist or https://www.filedropper.com/ and add the link here
+
+### Make sure all the checkboxes below are checked. If not, the issue will be deleted.
+<!--- The checkboxes can be checked AFTER the issue is submitted. -->
+- [ ] This issue isn't duplicated - check by using the search bar at the top of your screen, and selecting the "issues" tab.
+- [ ] This issue is written in english for clarity. Use https://translate.google.com to translate to english.
```
Comment (SOF3) [inline .github/ISSUE_TEMPLATE]: No need to advertise Google Translate.

File: .github/ISSUE_TEMPLATE [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+- [ ] This issue does not use abbreviations- type out the full word or phrase please. (Understandability)
```
Comment (SOF3) [inline .github/ISSUE_TEMPLATE]: Invalid point. It is very reasonable to use common abbreviations.

File: .github/PULL_REQUEST_TEMPLATE [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+### What does this fix?
+
+> Does this fix any problems with the server? Please post images of the issue, and the solution, or submit a link to the issue (eg: #11111111)
+
+### Make sure all the checkboxes below are checked. If not, the PR will be deleted.
+<!--- The checkboxes can be checked AFTER the PR is submitted. -->
+- [ ] This PR isn't duplicated - check by using the search bar at the top of your screen, and selecting the "issues" tab.
+- [ ] This PR is written in english for clarity. Use https://translate.google.com to translate to english.
+- [ ] This PR does not use abbreviations- type out the full word or phrase please. (Understandability)
+- [ ] This PR submits something useful
```
Comment (SOF3) [inline .github/PULL_REQUEST_TEMPLATE]: Same problems with issues.

File: README.md [hunk 7]
```diff
@@ -40,7 +40,7 @@ __PocketMine-MP is a free, open-source software that creates Minecraft: Pocket E
 [#pmmp + #pocketmine channel WebIRC](http://webchat.freenode.net/?channels=pmmp,pocketmine)
 
 ### Want to contribute?
-* Check the [Contributing Guidelines](CONTRIBUTING.md)
+* Check the [Contributing Guidelines](.github/CONTRIBUTING.md)
```
Comment (SOF3) [inline README.md]: I don't think moving CONTRIBUTING.md is necessary since it has existed here for ages.

File: .github/PULL_REQUEST_TEMPLATE [hunk 8]
```diff
@@ -0,0 +1,16 @@
+## **SUBMISSIONS LACKING A RESPONSE TO ONE OF THE FIELDS WILL BE DELETED**
+
+### Pull Request Description:
+
+> Write a short description about the Pull Request
```
Comment (SOF3) [inline .github/PULL_REQUEST_TEMPLATE]: This pull request template is poor. You have not covered all kinds of pull requests.

File: .github/ISSUE_TEMPLATE (hunk 3 above)
Comment (SOF3) [inline .github/ISSUE_TEMPLATE]: And an issue can be related to many things, not just core bugs. There are professional issues that are not related to the game at all.

**Comment by SOF3** [comment]
BTW, what's wrong with ISSUE_TEMPLATE.md instead of ISSUE_TEMPLATE?

**Comment by SOF3** [comment]
Actually "Multiple things, read description" already seriously violate the contribution guidelines.

File: .github/ISSUE_TEMPLATE (hunk 1 above)
Comment (remotevase) [inline .github/ISSUE_TEMPLATE]: Well, what CAN i use before the issue is not created?

File: .github/ISSUE_TEMPLATE (hunk 2 above)
Comment (remotevase) [inline .github/ISSUE_TEMPLATE]: I simply find it easier to give people the link for something as opposed to making them do the work themselves.

File: .github/ISSUE_TEMPLATE (hunk 4 above)
Comment (remotevase) [inline .github/ISSUE_TEMPLATE]: I simply find it easier to give people the link for something as opposed to making them do the work themselves.

File: .github/ISSUE_TEMPLATE (hunk 5 above)
Comment (remotevase) [inline .github/ISSUE_TEMPLATE]: mmm... I thought this would be controversial... It always depends on who owns the organization. Some people like abbreviations, some people don't.
